templates/preview.html

```diff
@@ -44,6 +45,7 @@ <h1 id="abe-preview-header-title" class="h3 center truncate caps">Demo: {{subHea
     {{^document.isAmpStory}}
     {{> analytics.html}}
     {{> serviceworker.html}}
+    {{> consent.html}}
```

Comment:
Can you add the amp-story-user consent component in here:

```
{{#isAmpStoryTag}}       
{{> analytics.html}}
{{/isAmpStoryTag}}
```

Reply:
Sure! Thanks; I wasn't sure what that was.